components/App/App.jsx

```diff
@@ -2,7 +2,7 @@ const React = require('react')
 const Filter = require('../Filter')
 const TaskList = require('../TaskList')
 
-class Main extends React.Component {
+class App extends React.Component {
```

Comment:
> There's main.js, snoozer.js, and app.jsx

I agree, it is confusing. I got rid of `snoozer.js` but the logic is now in `index.html` so it still is confusing. I was thinking that everything would start with the App component, even signed-out users. If the user is logged out it will render the login page, if they are logged in it will render the logged in page.

But I'm totally willing to change this if it ends up not feeling right.